### PR TITLE
avro binary encoding / decoding.

### PR DESCRIPTION
**Pull request by zmstone**

### CODE REVIEW

**Comment by id** [comment]
Bump version?

File: src/avro_binary_decoder.erl [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+%%%
+%%% Unless required by applicable law or agreed to in writing,
+%%% software distributed under the License is distributed on an
+%%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%%% KIND, either express or implied.  See the License for the
+%%% specific language governing permissions and limitations
+%%% under the License.
+%%%
+%%% @doc
+%%% Encodes Avro values to binary format according to Avro 1.7.5
```
Comment (id) [inline src/avro_binary_decoder.erl]: Decodes Avro values from binary format

File: src/avro_binary_decoder.erl [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+bytes(Bin) ->
+  {Size, Rest} = long(Bin),
+  <<Bytes:Size/binary, Tail/binary>> = Rest,
+  {Bytes, Tail}.
+
+block(Bin, ItemDecodeFun) -> block(Bin, ItemDecodeFun, _Acc = []).
+
+block(Bin, ItemDecodeFun, Acc) ->
+  {Count0, Rest} = long(Bin),
+  case Count0 =:= 0 of
```
Comment (id) [inline src/avro_binary_decoder.erl]: You parse trailing 0 as avro-binary-encoded integer but in encoder you use plain integer 0: https://github.com/klarna/erlavro/pull/4/files#diff-957b7df97c68b310ae983c3e82138ba4R107

Am I missing something?

Comment (id) [inline src/avro_binary_decoder.erl]: Oh well, zero is always zero, right?

Comment (zmstone) [inline src/avro_binary_decoder.erl]: yes, 0 is encoded as <<0>>.

**Comment by zmstone** [comment]
@id done! please review.

**Comment by id** [comment]
Great job 👍

**Pull request merged**